platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -0,0 +1,860 @@
+# TSS-Library
+
+## Summary
+
+Provide necessary pieces for signing messages using a TSS scheme.
```

Comment:
It is not only about a public API; it will provide at least 2 public APIs, multiple modules, implementations of those APIs, utility libraries in multiple languages, project-building logic, artifacts, and documentation.
I can use the word artifacts if that better describes the produced elements...